test/test.hs

```diff
@@ -2448,6 +2417,82 @@ instance Arbitrary (LitOnly (Expr EWord)) where
 instance Arbitrary (LitOnly (Expr Buf)) where
   arbitrary = LitOnly . ConcreteBuf <$> arbitrary
 
+-- GenWordSimplifyExpr
+newtype GenWordSimplifyExpr a = GenWordSimplifyExpr a
```

Comment:
```suggestion
newtype ZeroDepthWord = ZeroDepthWord (Expr EWord)
```

I don't think we need the type parameter here since we only wanna use this type to generate `Expr EWord`? I would also prefer a more descriptive name

Reply:
Yep, done, thanks!